docs/getting-started/first-autoscale.md

```diff
@@ -12,11 +12,13 @@ Let's run our "Hello world!" Service just one more time. This time, try the Knat
 curl http://hello.default.127.0.0.1.nip.io
 ```
 
-You can watch the pods and see how they ==scale to zero== after traffic stops going to the URL.
+You can watch the pods and see how they scale to zero after traffic stops going to the URL.
 ```bash
 kubectl get pod -l serving.knative.dev/service=hello -w
 ```
 
+!!! warning
+    It may take up to 2 minutes for your pods to scale down, pinging your service again will reset this timer.
```

Comment:
It may take up to 2 minutes for your pods to scale down. Pinging your service again will reset this timer.